code-review-trainer-service/CodeReviewProblems/EasyCodeReviewProblems.cs

```diff
@@ -0,0 +1,302 @@
+namespace code_review_trainer_service.CodeReviewProblems;
+
+// Static class containing easy-level code review problems
+public static class EasyCodeReviewProblems
+{
+    private static readonly Random _random = new Random();
+    
+    private static readonly string[] _problems = new string[]
+    {
+        // Problem 1: Bad variable names + confusing logic
+        @"public class Calculator
+{
+    public int x(int a, int b)
+    {
+        int z = a + b;
+        if (z > 10)
+        {
+            z = z - 1;
+        }
+        else
+        {
+            z = z + 1;
+        }
+        return z;
+    }
+}",
+        
+        // Problem 2: Unreachable code + bad formatting
+        @"public void ProcessOrder(bool isValid)
+{
+if(isValid==true){
+Console.WriteLine(""Processing order..."");
+return;
+Console.WriteLine(""Order processed successfully"");
+}
+else{
+Console.WriteLine(""Invalid order"");
+}
+}",
+
+        // Problem 3: Bad variable names + failing logic
+        @"public bool CheckAge(int age)
+{
+    int x = 18;
+    if (age > x)
+    {
+        return false; // Should return true for adults
+    }
+    return true;
+}",
+
+        // Problem 4: Bad formatting + confusing logic
+        @"public string GetDiscount(double price,int customerType){
+double d=0.0;
+if(customerType==1){d=0.1;}else if(customerType==2){d=0.15;}else{d=0.05;}
+return (price*d).ToString();
+}",
+
+        // Problem 5: Bad variable names + unreachable code
+        @"public void PrintNumbers()
+{
+    int i = 1;
+    while (i <= 5)
+    {
+        Console.WriteLine(i);
+        i++;
+        break;
+        Console.WriteLine(""This will never print"");
```

Comment:
Should not include "This will never print". Instead make it look like something that _should_ print. Like $"New value: {i}"